.github/workflows/ci.yml

```diff
@@ -47,11 +47,16 @@ jobs:
         trino: [
           "latest",
         ]
+        sqlalchemy: [
+          "1.4"
+        ]
         include:
           # Test with older Trino versions for backward compatibility
-          - { python: "3.10", trino: "351" }  # first Trino version
+          - { python: "3.10", trino: "351", sqlalchemy: "1.4" }  # first Trino version
           # Test with Trino version that requires result set to be fully exhausted
-          - { python: "3.10", trino: "395" }
+          - { python: "3.10", trino: "395", sqlalchemy: "1.4" }
+          # Test with sqlalchemy 1.3
+          - { python: "3.10", trino: "395", sqlalchemy: "1.3" }
```

Comment:
We can probably remove this now since the actual issue was fixed in the client itself (you can even say the client did not follow the protocol correctly).

(Do it as separate commit).

Reply:
Done